index.html

```diff
@@ -122,16 +137,20 @@ <h2> Tonic</h2>
 
       <div class="Card-Works">
         <div class="Snapshoot-Portfolio">
-          <img src="./media/multi-post.png" alt="Website">
+          <img src="./media/Snapshoot Portfolio-4.png" alt="Website">
         </div>
 
         <div class="Left-Block">
           <div class="Primary-Text">
             <h2> Multi-Post Stories</h2>
```

Comment:
- [ ] As requested by the previous reviewer, kindly increase the `font-size` and `font-weight` of the title of the cards to match the design on Figma (`40px` size and `700` weight)

| Your design   | Figma design  |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/50721479/199692793-13f53931-f0c5-456f-8ba6-da8369d852d3.png) | ![image](https://user-images.githubusercontent.com/50721479/199693050-f06c4516-2502-419c-894e-5db9b3717170.png)  |